### PR TITLE
Update Android sync section with table and subsections

### PR DESCRIPTION
**Pull request by ejohnso49**
### Summary

Made some changes based on support tickets we received. Main goal was to emphasize what the different parameters are used for

### CODE REVIEW

File: docs/android/android-bort.mdx [hunk 1]
```diff
@@ -528,20 +528,50 @@ collected from the system, it is uploaded up to 15 minutes later.
...
 (if true) or
 [NetworkType.UNMETERED](https://developer.android.com/reference/androidx/work/NetworkType#UNMETERED)
 constraint on its network requests.
 
-Metrics, logs, and DropBoxManager Entries are uploaded every 2 hours by default.
-OTAs are checked for every 12 hours. Both settings can be adjusted if desired by
-contacting support.
+#### Sync Rates {#sync-rates}
+
+The table below outlines the default rates for how often each type of Memfault
```
Comment (sjp4) [inline docs/android/android-bort.mdx:548]: I would actually probably revert this change - I'm not sure it makes sense to break out metrics/logs/dropbox when they are all uploaded as one `mar` bundle every 2 hours.

**Review by sjp4 — APPROVED** [review]

File: docs/android/android-bort.mdx (hunk 1 above)
Comment (ejohnso49) [reply docs/android/android-bort.mdx:548]: That's a good idea, I can do that! I can put up a change to summarize that
